cmd/generate.go

```diff
@@ -68,7 +68,27 @@ func newGenerateCmd() *cobra.Command {
 				return errors.Wrap(err, "loading API model in generateCmd")
 			}
 
-			return gc.run()
+			azurePublicCloudSpec := &datamodel.AzureEnvironmentSpecConfig{
```

Comment:
this looks exactly same with AzureCloudSpec, can we use that? Probably rename to something like AzureDefaultCloudSpec

Reply:
I just deleted AzureCloudSpec from its old location because we don't need it. cmd/generate is now the only code (other than test code) that constructs a cloud spec config.